docs/UserGuide.md

```diff
@@ -87,21 +87,24 @@ Format: `help`
 
 ### Adding a guest: `add`
 
-Adds a guest to the guest book.
+Adds a guest to the guest book. This is akin to checking a guest in with their personal and room details.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL rm/ROOM dr/DATE_RANGE ng/NUMBER_OF_GUESTS`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL rm/ROOM dr/DATE_RANGE ng/NUMBER_OF_GUESTS [rq/REQUEST]`
 
-[//]: # (Commented out first as not relevant since we do not have optional fields)
-[//]: # (But may be relevant in the future)
-[//]: # (<div markdown="span" class="alert alert-primary">:bulb: **Tip:**)
+Constraints of parameters:
 
-[//]: # (A guest can have any number of tags &#40;including 0&#41;)
-
-[//]: # (</div>)
+Unless stated otherwise, all fields should not be left blank.
```

Comment:
Should this be indented to show that it is under the Constraints of parameters?

![image](https://user-images.githubusercontent.com/87851444/197387310-032acfcd-a2ee-47d6-9904-cb557c72b15a.png)

Reply:
Edited!